docker/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM centos:7
+
+RUN yum install -y epel-release && yum install -y git python-pip 
+RUN pip install configparser && pip install gitpython requests future jinja2 colorama
+RUN mkdir /build
+WORKDIR /build
+ENTRYPOINT ["/build/build.sh"]
```

```suggestion
ENTRYPOINT ["sh", "/build/build.sh"]
```
It's safer to just have `sh` execute it instead of relying on the script specified shebang.

Another approach would be to make the shebang in the build.sh script to `#!/bin/sh`, i guess :)